lib/ansible/modules/cloud/docker/docker_volume.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/python
+# coding: utf-8
+#
+# Copyright 2017 Red Hat | Ansible, Alex Grönholm <alex.gronholm@nextday.fi>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = u'''
+module: docker_volume
+version_added: "2.4"
+short_description: Manage Docker volumes
+description:
+  - Create/remove Docker volumes.
+  - Performs largely the same function as the "docker volume" CLI subcommand.
+options:
+  name:
+    description:
+      - Name of the volume to operate on.
+    required: true
+    aliases:
+      - volume_name
+
+  driver:
+    description:
+      - Specify the type of volume. Docker provides the C(local) driver, but 3rd party drivers can also be used.
+    default: local
+
+  driver_options:
+    description:
+      - >
+        Dictionary of volume settings. Consult docker docs for valid options and values:
+        U(https://docs.docker.com/engine/reference/commandline/volume_create/#driver-specific-options)
+    default: null
+
+  force:
+    description:
+      - With state I(present) causes the volume to be deleted and recreated if the volume already
+        exist and the driver, driver options or labels differ.
+        This will cause any data in the existing volume to be lost.
+    default: false
+
+  state:
+    description:
+      - I(absent) deletes the volume.
+      - I(present) creates the volume, if it does not already exist.
+    default: present
+    choices:
+      - absent
+      - present
+
```

Comment:
I don't see the debug-option mentioned in the documentation.
Is that on purpose ?